app/models/lsg_body.rb

```diff
@@ -10,4 +10,12 @@ class LsgBody < ApplicationRecord
          corporation: 'Corporation',
          panchayat: 'Panchayat',
        }
+
+  def archive!
+    update(archived: true)
+  end
+
+  def unarchive!
+    update(archived: false)
+  end
```

Comment:
This can be in the controller itself as we aren't using this method anywhere else. It's good to add a scope instead that will list live LSG's

Reply:
Yes. Moved to the controller. Have added a scope to list live lsgs.